pkg/crc/config/settings.go

```diff
@@ -27,7 +27,6 @@ const (
 	ProxyCAFile             = "proxy-ca-file"
 	ConsentTelemetry        = "consent-telemetry"
 	EnableClusterMonitoring = "enable-cluster-monitoring"
-	AutostartTray           = "autostart-tray"
```

Comment:
?

looks unrelated

Reply:
@gbraad different commit.

Reply:
Still, this could be a PR that would have been merged instantly